README.md

```diff
@@ -118,6 +118,82 @@ For `RelatedProducts`:
 | `summary`                   | `Object`   | Product Summary schema properties.                        | -
 | `gap`                       | `Enum`   | Gap between items. Possible values: `ph0`, `ph3`,`ph5`, `ph7`| `ph3`
 
+For `TabbedShelf`:
+
+| Prop name          | Type       | Description   | Default value |
+| ------------------ | ---------- | ------------------------------------------------------------------------ | ---------------- |
+| `isEnabled`            | `Boolean`   | True if component should appear | `false`
+| `headline`            | `String`   | Text shown on top of component (leave empty to not display) | `''`
+| `bottomText`            | `String`   | Text shown on bottom of component (leave empty to not display) | `''`
+| `buttonText`            | `String`   | Text shown on button underneath the component (leave empty to not display the button) | `''`
+| `buttonUrl`            | `String`   | Url to redirect user when button is pressed | `''`
+| `tabs`               | `TabsSchema` | Tabs configuration to be displayed.  `See TabsSchema`          | -
+| `shelf`               | `Shelf` | Props for the shelf displayed, same type as `Shelf`.  `See Shelf`          | -
+
+For `TabsSchema`:
+| Prop name          | Type       | Description   | Default value |
+| ------------------ | ---------- | ------------------------------------------------------------------------ | ---------------- |
+| `children`            | `[TabsChildrenSchema]`   | Array of children to be displayed. Maximum number of items is 6. Items should be of `TabsChildrenSchema`  | `-`
+
+For `TabsChildrenSchema`:
+| Prop name          | Type       | Description   | Default value |
+| ------------------ | ---------- | ------------------------------------------------------------------------ | ---------------- |
+| `id`            | `Number`   | id of category to be displayed  | `-`
+
+Example of Usage of `TabbedShelf`:
```

Comment:
"Since `TabbedShelf` props have a different structure, we add an example of usage below: "